src/components/ComboBox/_combo-box.scss

```diff
@@ -29,16 +15,49 @@
     }
   }
 
+  // Placement of the editOptionText
   &.#{$iot-prefix}--combobox-add {
-    .#{$prefix}--list-box__menu-item:first-child {
-      &::after {
-        content: attr(data-text);
-        color: $text-03;
-        position: absolute;
-        right: $spacing-03;
-        top: 50%;
-        transform: translateY(-50%);
-      }
+    $ai-apps-combobox-input-height-sm: $spacing-07;
```

Comment:
These are cool. Are they variables in the JS?

Reply:
No, just something I made in the sass so it's a bit easier to read and understand what I'm doing